azure-pipelines.yml

```diff
@@ -4,11 +4,38 @@ pr:
 pool:
   vmImage: 'windows-latest'
 
-steps:
-- task: DotNetCoreCLI@2
-  inputs:
-    command: 'build'
-    projects: src/SarifWeb.sln
-    configuration: 'Release'
-
-- task: ComponentGovernanceComponentDetection@0
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+extends:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
+    parameters:
+      pool:
+        name: 1ES-Shared-Hosted-Pool_Windows-Server-2022
+        os: windows
+      containers:
+        default_windows_container:
+          image: mcr.microsoft.com/windows/servercore:ltsc2022
+        
+      stages:
+      - stage: build
+        jobs:
+        - job: build
+
+
+          steps:
+          - task: UseDotNet@2
+            displayName: 'Use .NET Core sdk'
+            inputs:
+              packageType: sdk
+              version: 5.0.x
+          - task: DotNetCoreCLI@2
+            inputs:
+              command: 'build'
+              projects: src/SarifWeb.sln
+              configuration: 'Release'
+
+          - task: ComponentGovernanceComponentDetection@0
```

Comment:
Is this task required? Check if 1ES perform that validation

Reply:
removed, as this is not required